pkg/platform/kube/platform_test.go

```diff
@@ -1146,6 +1146,145 @@ func (suite *FunctionKubePlatformTestSuite) TestAlignIngressHostSubdomain() {
 	}
 }
 
+func (suite *FunctionKubePlatformTestSuite) TestEnrichFunctionWithPreemptionSpec() {
```

Comment:
Missing tests for prevent and indeed it seems like logic there is missing some pruning

Reply:
pruning was not missing, it was done by overriding the common ancestor field and thus pruning